src/__mocks__/react-i18next.js

```diff
@@ -1,4 +1,8 @@
 import React from 'react'
 
 // this mock makes sure any components using the translate HoC receive the t function as a prop
-export const translate = (key) => Component => props => <Component t={() => key} {...props} />
+export const translate = (namespace) => Component => {
+  const Translated = props => <Component t={key => `${namespace}:${key}`} {...props} />
+  Translated.displayName = `Translate(${Component.displayName || Component.name})`
```

Comment:
I think it's maybe good to do a check for production here. Not sure what exactly the benefit is but major libraries do it:
https://github.com/acdlite/recompose/blob/c830f184123eebc8ea6b8b712840fdc86a680fa2/src/packages/recompose/getContext.js#L15

Reply:
I don't see why we should do that.

Reply:
k, me neither :D
You can refactor this though to:
```
setDisplayName(wrapDisplayName(Component, 'Translate'))(Translated)
```

Not sure if we want to use this library that heavily though.